x-pack/plugins/observability_onboarding/public/routes/templates/system_logs.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useBreadcrumbs } from '@kbn/observability-shared-plugin/public';
+import React, { ComponentType, useRef, useState } from 'react';
+import { breadcrumbsApp } from '../../application/app';
+import { Provider as WizardProvider } from '../../components/app/system_logs';
+import {
+  FilmstripFrame,
+  FilmstripTransition,
+  TransitionState,
+} from '../../components/shared/filmstrip_transition';
+
+interface Props {
+  children: React.ReactNode;
+}
+
+export function SystemLogs({ children }: Props) {
+  useBreadcrumbs(
+    [
+      {
+        text: i18n.translate(
+          'xpack.observability_onboarding.breadcrumbs.systemLogs',
+          { defaultMessage: 'System logs' }
+        ),
+      },
+    ],
+    breadcrumbsApp
+  );
+  return <AnimatedTransitionsWizard>{children}</AnimatedTransitionsWizard>;
+}
+
+const TRANSITION_DURATION = 180;
+
+function AnimatedTransitionsWizard({ children }: Props) {
```

Comment:
I think this could be pretty much reused, the only significant change I see it's just the title. wdyt?

Reply:
I went ahead and just removed the animated transition entirely. Since there's only one step, there's nothing to animate anyways.